amazon-kinesis-client/src/test/java/software/amazon/kinesis/coordinator/SchedulerTest.java

```diff
@@ -641,6 +643,7 @@ public ShardSyncTaskManager createShardSyncTaskManager(MetricsFactory metricsFac
             shardSyncTaskManagerMap.put(streamConfig.streamIdentifier(), shardSyncTaskManager);
             shardDetectorMap.put(streamConfig.streamIdentifier(), shardDetector);
             when(shardSyncTaskManager.shardDetector()).thenReturn(shardDetector);
+            when(shardDetector.streamIdentifier()).thenReturn(streamConfig.streamIdentifier());
```

Comment:
Is there test cases added for multistream in the scheduler ?

Reply:
Will send with updated CR